fv3core/fv3core/testing/translate.py

```diff
@@ -293,6 +296,12 @@ def make_grid_storage(self, pygrid):
             if k in self.data:
                 self.make_composite_var_storage(k, self.data[k], shape)
                 del self.data[k]
+        for k in TranslateGrid.ee_vars:
```

Comment:
nit: This was pre-existing, but can you replace `k` here and below with `key`? When I read these my mind doesn't stop telling me it's indexing on the vertical direction.

Reply:
sure thing